parser/model/model.go

```diff
@@ -1326,6 +1326,16 @@ func (index *IndexInfo) HasPrefixIndex() bool {
 	return false
 }
 
+// HasColumn checks whether the index contains the column with the specified ID.
+func (index *IndexInfo) HasColumn(tblInfo *TableInfo, colID int64) bool {
```

Comment:
Could we unify this function name and `findColumnInIndexCols`？One is filtered by ID, another one is filtered by name.

Reply:
Updated.